src/java/picard/analysis/artifacts/SequencingArtifactMetrics.java

```diff
@@ -212,7 +228,8 @@ public void calculateDerivedStatistics() {
         public double REV_ERROR_RATE;
 
         /**
-         * The bait bias error rate, calculated as max(1e-10, FWD_ERROR_RATE - REV_ERROR_RATE).
+         * The estimated error rate due to this artifact.
+         * Calculated as max(1e-10, (PRO_ALT_BASES - CON_ALT_BASES) / (PRO_ALT_BASES + PRO_REF_BASES + CON_ALT_BASES + CON_REF_BASES)).
```

Comment:
I shouldn't have changed this comment, it happened by accident when I thought I would refactor out the common part from BaitBiasDetailMetrics and PreAdapterDetailMetrics. I didn't do that in the end due to lack of control over the order the fields of the resulting class.